standard_releases.md

```diff
@@ -1,59 +1,215 @@
+the translation of each as a separate project. In this case, it's probably most
+helpful to translate the current release first, bring across any fixes and only
+initiate translation of `master` when it's due for release.
+
+### Fixes to an existing release
+
+When a bug is discovered or a correction needs to be made to an existing
+release and it is deemed severe enough to require an immediate fix (i.e.
+it cannot wait for the next naturally-occurring release), then we need
+to make an unplanned release. The process for this is as follows:
+
+* Make a new branch named something like `v0-2-0-description-of-issue`
+  from the current release branch and commit the fix(es) to it
+* Review and test this branch in a pull-request
+* If necessary, perform necessary translation work on this branch once this fix
+  is approved in English. Whether this is necessary depends on the nature of the
+  fix, i.e. whether it changes translated text.
+* At this point, we need to decide on the impact of the changes on users of the
+  standard and the appropriate new version number. Create a new release branch
+  named accordingly, e.g. v0-2-1 or v0-3-0 and merge the fixes into it.
+* Follow the usual release process of enabling, updating to latest, etc, so that
+  this release becomes the new 'current' release. If the issue warrants it,
+  consider adding a notice to the old release documenting the issue and that it
+  should not be used. You may also have to notify existing users of that
+  version.
+* If the fixed issue is also present in `master`, the fix branch should also be
+  merged back into there, in a separate pull-request.
+
+**Note:** until we hit a stable 1.0 release, only very severe bugs that impede real
+world use of the standard, present a security risk or otherwise cause significant
+damage to our aims as an organisation should require this kind of fix. It is also
+assumed that other older pre 1.0 releases are automatically and immediately
+considered end-of-life when a new version is released. This policy is likely to
+change only when we have real world use or commitments mandating a greater level
+of support.
+
+### Fixes to an existing translation
+
+Fixes that solely relate to an existing translation (i.e. not the source text,
+but a particular translation of it) should follow a similar process to fixes to
+an existing release, except that they should not require a change in version
+number or a new release branch. The fix branch should be merged directly back
+into the existing release branch.
+
+### Non-material changes
+
+Changes which do not affect the standard, or normative documentation accompanying
+it, do not need a corresponding change in version number. However, it's still
+important to consider whether any text has been added or changed, and which
+existing versions of the standard you wish the changes to appear in.
+
+**Examples:**
+
+- A change to the visual style of the standard website such as
```

Comment:
This example will also need

> update requirements.txt in the data-standard-sphinx-theme so that https://github.com/openownership/data-standard/blob/master/requirements.txt#L8 refers to the latest commit

Reply:
I've mentioned that in 3d20f15, thanks!